src/main/java/org/sagebionetworks/web/client/widget/entity/file/ProjectTitleBarView.java

```diff
@@ -0,0 +1,18 @@
+package org.sagebionetworks.web.client.widget.entity.file;
+
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import org.sagebionetworks.repo.model.EntityType;
+import org.sagebionetworks.web.client.SynapseView;
+
+public interface ProjectTitleBarView extends IsWidget, SynapseView {
```

Comment:
`ProjectTitleBarView` is the same as the previous implementation of `BasicTitleBarView`--just a rename.